src/convertToObject.js

```diff
@@ -13,6 +13,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const cssStyles = {};
+
+  const cssPropertiesAndValues = sourceString
+    .split(';')
+    .map(propertyOrValue => propertyOrValue.split(':'))
+    .filter(propertyOrValue => propertyOrValue.length === 2);
+
+  for (const propertyOrValue of cssPropertiesAndValues) {
+    cssStyles[propertyOrValue[0].trim()] = propertyOrValue[1].trim();
```

Comment:
Try [array destructuring](https://www.freecodecamp.org/news/array-destructuring-in-es6-30e398f21d10/) here, you will avoid using mysterious values like property[0]